src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -596,6 +599,7 @@ def load_arguments(self, _):
                    default=os.path.join(os.path.expanduser('~'), '.kube', 'config'))
         c.argument('public_fqdn', default=False, action='store_true')
         c.argument('credential_format', options_list=['--format'], arg_type=get_enum_type(credential_formats))
+        c.argument('namespace_name', options_list=['--namespace'], help='If specified, the credentials are returned at the namespace scope, assuming the user has access to the ARM resource for that namespace.')
```

Comment:
User only having access to namespace resource can use this parameter to specify the namespace

Reply:
Done